Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHost.cs

```diff
@@ -41,15 +41,15 @@ public string InitialTypeName
         /// </summary>
         /// <remarks>This UWP XAML element is the root element of the wrapped DesktopWindowXamlSource.</remarks>
         [Browsable(true)]
-        public windows.UI.Xaml.UIElement Child
+        public WUX.UIElement Child
         {
             get => ChildInternal;
 
             set => ChildInternal = value;
         }
 
         /// <summary>
-        /// Creates <see cref="windows.UI.Xaml.Application" /> object, wrapped <see cref="windows.UI.Xaml.Hosting.DesktopWindowXamlSource" /> instance; creates and
+        /// Creates <see cref="WUX.Application" /> object, wrapped <see cref="WUX.Hosting.DesktopWindowXamlSource" /> instance; creates and
```

Comment:
i'm curious, does this `<see cref="..." />` create a link to the metadata associated with the type?

Reply:
I think it does, so perhaps this replacement is best left as it was

Reply:
Build will fail, doc tool needs to be able to resolve types.

---
In reply to: [260108578](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260108578) [](ancestors = 260108578)

Reply:
Yes.

---
In reply to: [260106667](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r260106667) [](ancestors = 260106667)